### DISCUSSION TITLE
Custom utility classes does not include in tailwind build css

### DISCUSSION THREAD

**Discussion by shahaab-ali** [tailwindlabs/tailwindcss#9266] [Help]
Hi, 

Can anyone help? Custom utility classes does not include in tailwind build CSS?

**Comment by wongjn** [comment]
How have you added your custom utility classes? If you could share some of your code, we can have a better look at what might be going wrong =)

**Reply by shahaab-ali** [reply]
@wongjn

 @tailwind base;
@tailwind components;
@tailwind utilities;

@layer utilities {
    .text-shadow {
        text-shadow: -1px 0 #52B4E0, 0 1px #52B4E0, 1px 0 #52B4E0, 0 -1px #52B4E0;
    }

    .text-outline {
        text-shadow: 2px 2px 0 #4074b5, 2px -2px 0 #4074b5, -2px 2px 0 #4074b5, -2px -2px 0 #4074b5, 2px 0px 0 #4074b5, 0px 2px 0 #4074b5, -2px 0px 0 #4074b5, 0px -2px 0 #4074b5;
        color: rgb(16 37 56);
    }
}

.tab.tab-active {
    @apply border-blue-400
}

**Reply by wongjn** [reply]
So `text-shadow` or `text-outline` is not appearing in your final built CSS? Do they appear anywhere in the files covered by your `content` globs in your Tailwind config? Since they are in a Tailwind `@layer`, they won't appear in your final built CSS if they do not appear in any content scanned by Tailwind.

**Reply by shahaab-ali** [reply]
Yes, I have used it in index.html, and tailwind also generated these classes in output.css but these classes do not appear in the final built CSS

**Reply by wongjn** [reply]
Is `output.css` not your final built CSS? You have some kind of step after building with Tailwind?

**Reply by shahaab-ali** [reply]
output.css is just compiled CSS of tailwind classes. lastly, I want to create build CSS using `npx tailwindcss -o build.css --minify` text-outline and text-shadow is not appear in build.css
